packages/atomic/src/components/facets/facet-value/facet-value.tsx

```diff
@@ -0,0 +1,44 @@
+import {Component, Event, EventEmitter, h, Prop} from '@stencil/core';
+import {randomID} from '../../../utils/utils';
+
+@Component({
+  tag: 'facet-value',
+  shadow: false,
+})
+export class FacetValue {
+  @Event() public facetValueSelected!: EventEmitter<void>;
+  @Prop()
+  public label!: string;
+  @Prop()
+  public numberOfResults!: number;
+  @Prop()
+  public isSelected!: boolean;
+
+  public render() {
+    const id = randomID('');
```

Comment:
The string argument can be used to pass a prefix to the id, for example `input`. It would be better to do this rather than repeat `-input` in multiple places below.

If it must be a suffix, then we could do:

```
const id = `${randomID('')}-input`
```